web/config/defaultSettings.ts

```diff
@@ -31,6 +31,14 @@ export default {
   title: 'APISIX Dashboard',
   pwa: false,
   iconfontUrl: '',
+  serveUrlMap: {
+    dev: 'http://139.217.190.60',
+    test: 'http://localhost:9000',
+  },
```

Comment:
I found that both fe dev and cy test need this config, I moved it to the `defaultSettings`